src/main/java/seedu/address/logic/commands/cookbook/CookbookListCommand.java

```diff
@@ -22,7 +23,7 @@
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-
+        model.sortCookbook(new RecipeDefaultComparator());
```

Comment:
Does this resolve the bug of `cookbook search inventory` overwriting the alphabetical order of the default recipe list?

Reply:
yep, it sorts the cookbook by alphabetical order before listing.